website/content/resources/videos/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: TAG Contributor Strategy Videos
-linkTitle: Videos
+linkTitle: TAG-CS Videos
```

Comment:
What's the reason for changing this linktitle?

Reply:
<img width="1031" alt="image" src="https://github.com/cncf/tag-contributor-strategy/assets/1394552/5ee0936b-6434-4c39-bc22-34608e93f5d7">
I was trying to be nice and make the page consistent before I started adding more stuff to it 😸